### ISSUE TITLE
Taxon Details page

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/portal#28]
https://www.dev.catalogue.life/data/taxon/6e6323b3-399e-44b5-b94c-0d3a071ed30d

![image](https://user-images.githubusercontent.com/28574252/93245285-b8514680-f750-11ea-9a91-90678334b529.png)

Essential corrections needed:

- [x] Please place Classification after Synonyms (now it follows after Remarks)
**Better order for data:**
Accepted name
Synonyms & Combinations
Common names
Place in the classification
Distribution
Environment
Additional data
Source database
Recommended bibliographic citation
Link to original resource

- [x] Pls change heading Life Zone into **Environment**. (Life Zone is incorrect, preoccupied term: https://en.wikipedia.org/wiki/Life_zone) 
In old interface it is Environment: http://www.catalogueoflife.org/annual-checklist/2019/details/species/id/e6912cb265e9921176744eab5bd10a5f

**Environment** is better than presently implemented "Environment(s)".

- [x] **Link to original resource** is missing (for species, it is URL to species page). This is very important part of CoL as a gateway to original resources.

- [ ] **Recommended bibliographic citation** is missing.

- [x] Heading Remarks (instead of **Additional Data**) does not reflect contend of the field ("Climbing: Not climbing; Conservation Status: Not Threatened; Habit: Herb; Lifespan: Perennial" - looks like additional data rather remarks).

- [x] Please use Alias + FullGSDName (ILDIS: World Database of Legumes) or only Alias (ILDIS) in Source database field

- [ ] Important interface philosophy: it would be better to add a name of this form in top heading as **Taxon Details**, instead of using taxon name as heading (Trifolium ambiguum M.Bieb.). 
Basic data in CoL interface were structured in: _Taxon Details_ pages and _Database Details_ pages, plus data access tools, such as _Browse_ and _Search_.

**Comment by mdoering** [comment]
I agree to most of the above proposal, just a few changes
 - **changed order**. But I would put the classification above vernacular names which can be very large.
 - **lifezone/environment**: we do use the term lifezone in all our 3 main exchange formats [ColDP](https://github.com/CatalogueOfLife/coldp/blob/master/README.md#lifezone), [ACEF](https://user-images.githubusercontent.com/327505/93311006-bdbc9880-f805-11ea-99ac-3871aae8fb26.png) and DwC. We use it in the [API](http://api.catalogue.life/vocab/lifezone). It seems very strange to use a different term in the UI. Lets please keep lifezone. Environment is not unoccupied either.

And some additions:

**extinct**
Extinct is currently given in a row on its own before the synonymy. I would prefer to shorten the boolean information and instead show extinct by adding the dagger symbol in front of the accepted name. I would simply add that to the label generated by the backend, so the UI doesnt have to show anything for extinct

**taxonomic status, rank**
Taxonomic status and rank are shown in onw row but 2 columns currently before the synonymy. This is a little cumbersome and the 2 columns do not read well. I would prefer if we can show these 2 rather short informations more compact as a single value, e.g. Status: `accepted species` or `provisionally accepted genus`. We never deal with synonyms on this page, so we only have the 2 values for the status.

**Comment by dhobern** [comment]
Environment is not too bad - we just mean marine/brackish/freshwater/terrestrial - I'd make it consistent everywhere.

I prefer the "accepted species" etc. form to the split shown in the image.

**Comment by mdoering** [comment]
Ok, then lets change lifeZone to environment everywhere!

**Comment by thomasstjerne** [comment]
> Important interface philosophy: it would be better to add a name of this form in top heading as Taxon Details, instead of using taxon name as heading (Trifolium ambiguum M.Bieb.).
Basic data in CoL interface were structured in: Taxon Details pages and Database Details pages, plus data access tools, such as Browse and Search.

I´ll adapt to this - But I do disagree. I would definitely prefer the heading of a taxon page to be the name of that taxon rather than spelling out the obvious (that I am looking at a taxon page).

**Comment by mdoering** [comment]
>> Important interface philosophy: it would be better to add a name of this form in top heading as Taxon Details, instead of using taxon name as heading (Trifolium ambiguum M.Bieb.).
Basic data in CoL interface were structured in: Taxon Details pages and Database Details pages, plus data access tools, such as Browse and Search.
> I´ll adapt to this - But I do disagree. I would definitely prefer the heading of a taxon page to be the name of that taxon rather than spelling out the obvious (that I am looking at a taxon page).

I also strongly disagree. @thomasstjerne please keep the heading to be the name!
Taxon Details does not give you any information what the page is about. Similarily the dataset details page uses the dataset title not "Dataset Details"

**Comment by thomasstjerne** [comment]
OK now both versions exist in code, and for now I commented out the "Dataset details" and "Taxon details" headers in favour of the names

**Comment by mdoering** [comment]
To summarise the planned changes of the order and exact labels to be used in the taxon details page:

 - [ ] accepted name stays in title as it is
 - [ ] Status: accepted species (status & rank)
 - [ ] Synonyms
 - [ ] Classification
 - [ ] Common names
 - [ ] Distribution
 - [ ] Environment
 - [ ] Remarks
 - [ ] Source dataset: source dataset alias as link
 - [ ] Online resource: taxon.link
 - [ ] Recommended citation



TO BE AGREED STILL IN TODAYS CALL:
 - Add the extinct dagger in front of all extinct taxa in the Taxon.label
   - Does that also apply to all synonyms?
 - Accept the term "source dataset" everywhere instead of source database, contributing dataset, constituent, provider, GSD, etc. The API uses this term and I believe this is the best and most neutral one we have so far
 - How to generate the recommended citation

**Comment by yroskov** [comment]
**Added on 2020-10-06:**

https://www.dev.catalogue.life/data/taxon/9ce7b231-1370-4119-a9f3-2c3a561d6260

- [ ] Two fields “Status” and “Nomenclatural Status” look confusing.
It might be better to use heading “_Checklist Status_” instead of “Status” with values Accepted order (Provisionally accepted order) vs Synonym. I guess, synonyms (if any) will be listed below accepted name, as they appear on species page.

- [ ] _Nomenclatural Status_: acceptable. I guess, it should be, at least, “accepted”. 
Well, only statuses mentioned in Codes should appear in this field (ICZN: “valid”, “synonym”, “available”, “unavailable” etc.; different statuses are in botanical code. There is no such nomenclatural status as “accepted” in any Code).

- [ ] Change field heading “_Includes_” to “_Statistics_”. It’s better to use name of ranks in plural, e.g. Phyla, Classes, Orders, Families, Genera, Species

- [ ] _Vernacular names_. Column headings in the table should be: Original name, Transliterated name, Language, Country of use

**Comment by mdoering** [comment]
@thomasstjerne some of the tasks in the summary above have not been done yet. 
@yroskov the taxonomic status was supposed to be presented together with the rank.
The nomenclatural status is indeed tricky to visualize as it depends on the nomenclatural code. We only have zoological and botanical terms in the API: http://api.catalogue.life/vocab/nomstatus @thomasstjerne can we render those terms based on the code and maybe default to zoology in case no code is given? @yroskov @gdower we should make sure we have a code assigned to all taxa. I had proposed to [inherit the code from its parents](https://github.com/CatalogueOfLife/backend/issues/377), i.e. the kingdom ultimately.

**Comment by yroskov** [comment]
Regarding data for the field Nomenclatural Status. I would like to see statuses in CoL exactly as they are present in source GSDs - without interpretation by the software, or by Geoff and me on stage of data modification in CoLDP. 

As soon as we collect critical mass of nom statuses in CoL, we'll do analysis and formalization.

**Comment by mdoering** [comment]
@yroskov we can change the status enum later on if we really need to.
But there went many hours if not days into this and I am reluctant to change the whole setup now a few days before we want to go public. There has been more than a critical mass already

**Comment by mdoering** [comment]
> Recommended citation

Currently the recommended citation for a taxon is simply the source dataset citation: 
http://www.catalogueoflife.org/col/details/species/id/12dca9c49741815f82400bb7bff50553

This seems very odd. There should be some notion of the species name or identifier at least. I guess we want to have a DOI in that place at some stage. @dhobern @olafbanki @yroskov I would postpone this for a later stage after we had a discussion what this should exactly be. In particular as the [citation model is rather prominent in the new portal pages](https://www.dev.catalogue.life/content/colcitation.html). I have created a new issue just for that: https://github.com/CatalogueOfLife/portal/issues/42

**Comment by yroskov** [comment]
Citation of the source GSD in the CoL with each taxon borrowed from GSD is essential part of CoL social infrastructure. We must cite and promote our data providers. Use of DOI is a good idea, but each DOI should point to well structured bibliographic citation, which might be copy&paste to any kind of document created by CoL user.

**Comment by yroskov** [comment]
If we cannot implement DOIs right now, old style of bibliographic citation should be used until DOIs are ready.

GSD custodians might be very disappointed to see a new portal with less credits to their contribution, then previous product done.

**Comment by yroskov** [comment]
> 
> 
> @yroskov we can change the nom status enum later on if we really need to.
> But there went many hours if not days into this and I am reluctant to change the whole setup now a few days before we want to go public. There has been more than a critical mass already and it has been open for review for over a year already

I don't keep October as a deadline in my mind. But it should be done before we release CoL to public via new portal as a final product.

**Comment by mdoering** [comment]
Closing, only open discussion is on the taxon citation to be continued on #42
